mm2src/coins/utxo/rpc_clients/electrum_rpc/connection.rs

```diff
@@ -177,7 +177,7 @@ impl ElectrumConnection {
             settings,
             tx: Mutex::new(None),
             establishing_connection: AsyncMutex::new(()),
-            responses: Mutex::new(JsonRpcPendingRequests::new()),
+            responses: Mutex::new(JsonRpcPendingRequests::new_with_map_kind(MapKind::BTreeMap)),
```

Comment:
Picked `BTreeMap` intentionally to process entries in order.

Reply:
wdym by processing entries in order? what processing?

Reply:
> wdym by processing entries in order? what processing?

Making sure to process responses in the FIFO way.

Reply:
yeah where is that :joy:

Reply:
aha cool, im fine with BTree or others, i only nitted to make sure im not missing something with the in-order comment.